.gitignore

```diff
@@ -112,3 +112,6 @@ core.*
 *.log !run.log
 *.pyc
 Depends
+
+# ssr
+.vscode/
```

Comment:
- [ ] Minor thing: I just noticed this: I'm fine with this being included in the PR, but can you get rid of your initials here and move it up to with the other "editor files" ignore patterns?

Reply:
Sure, done (b89ea27).

Reply:
Thanks!